pyart/aux_io/gamic_hdf5.py

```diff
@@ -277,8 +287,17 @@ def _get_instrument_params(gfile, filemetadata):
     instrument_params['radar_beam_width_v'] = dic
 
     dic = filemetadata('pulse_width')
-    dic['data'] = gfile.sweep_expand(
-        gfile.how_attrs('pulse_width_us', 'float32') * 1e-6)
+    pw_names = ['pulse_width_us', 'pulse_width_mks', 'pulse_width']
+    pw_name = 'pulse_width_us'
+    for pw_name in pw_names:
+        if gfile.is_attr_in_group('/scan0/how', pw_name):
+            break
+    if pw_name == 'pulse_width':
+        dic['data'] = gfile.sweep_expand(
+        [0.2, 0.5, 1.0, 2.0][gfile.how_attrs(pw_name, 'int')] * 1e-6)
```

Comment:
With the latest version of NumPy this line is raising DeprecationWarning:

```
/home/jhelmus/dev/pyart/pyart/aux_io/gamic_hdf5.py:297: DeprecationWarning: converting an array with ndim > 0 to an index will result in an error in the future
  [0.2, 0.5, 1.0, 2.0][gfile.how_attrs(pw_name, 'int')] * 1e-6)
```
Selecting the first element of the extracted slice solves this:
```
[0.2, 0.5, 1.0, 2.0][gfile.how_attrs(pw_name, 'int')[0]] * 1e-6)
```